### PR TITLE
Fix IceKR, ComicFX, add Multiple Websites

### PR DESCRIPTION
**Pull request by MikeZeDev**
Fixes 
https://github.com/manga-download/hakuneko/issues/5112
https://github.com/manga-download/hakuneko/issues/5127
https://github.com/manga-download/hakuneko/issues/5102 
https://github.com/manga-download/hakuneko/issues/5100
https://github.com/manga-download/hakuneko/issues/5096
https://github.com/manga-download/hakuneko/issues/5071
https://github.com/manga-download/hakuneko/issues/5094
https://github.com/manga-download/hakuneko/issues/5039
https://github.com/manga-download/hakuneko/issues/4856
https://github.com/manga-download/hakuneko/issues/4820
https://github.com/manga-download/hakuneko/issues/5140

### CODE REVIEW

**Comment by MikeZeDev** [comment]
Fixes 
https://github.com/manga-download/hakuneko/issues/5112
https://github.com/manga-download/hakuneko/issues/5127
https://github.com/manga-download/hakuneko/issues/5102 
https://github.com/manga-download/hakuneko/issues/5100
https://github.com/manga-download/hakuneko/issues/5096
https://github.com/manga-download/hakuneko/issues/5071
https://github.com/manga-download/hakuneko/issues/5094
https://github.com/manga-download/hakuneko/issues/5039
https://github.com/manga-download/hakuneko/issues/4856
https://github.com/manga-download/hakuneko/issues/4820
https://github.com/manga-download/hakuneko/issues/5140

**Comment by 27136773** [comment]
Check and fix `lint` errors

File: src/web/mjs/connectors/MangaRolls.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class KawaScans extends WordPressMadara {
```
Comment (27136773) [inline src/web/mjs/connectors/MangaRolls.mjs]: Class name does not match file name, suggest to change so they are in sync

**Comment by 27136773** [comment]
Some icons for new added websites are missing, i suggest to add a default icon to prevent warnings in the console (e.g. as done for [antisensescans](https://github.com/manga-download/hakuneko/blob/master/src/web/img/connectors/antisensescans?raw=true))

**Review by knapah — APPROVED** [review]

File: src/web/mjs/connectors/MangaRolls.mjs (hunk 1 above)
Comment (MikeZeDev) [reply src/web/mjs/connectors/MangaRolls.mjs]: Fixed.

**Review by 27136773 — APPROVED** [review]

**Pull request merged**